guides/web-analytics.md

```diff
@@ -0,0 +1,41 @@
+# Web analytics
+
+We use Google Analytics on most of our sites. It's free ([within limits](https://support.google.com/analytics/answer/1070983?hl=en)), it's easy to implement and there aren't many things it can't track.
+
+The login details for access are kept in 1Password, search for "Google Analytics".
+
+## Implementing
+
+We add Google Analytics via [gtag.js](https://developers.google.com/analytics/devguides/collection/gtagjs/). If you're using [whippet-theme](https://github.com/dxw/whippet-theme), the code is already included and you just have to enter the correct tracking ID.
+
+### Event tracking
```

Comment:
This is exactly what I need. I would have otherwise I assumed this would be automatically available after plugging it in :+1: